src/main/java/org/isf/utils/file/FileTools.java

```diff
@@ -119,7 +119,6 @@ public static Date getTimestamp(File file) {
 					datesFound.add(date);
 				}
 			} catch (ParseException e) {
-				date = null;
```

Comment:
maybe we can at least log that no dates patterns have been found in the filename

Reply:
Added logging message.

Reply:
Thanks!